wire/io.go

```diff
@@ -0,0 +1,846 @@
+package wire
+
+import (
+	"github.com/evergreen-ci/mrpc/mongowire"
+	"github.com/mongodb/jasper"
+	"github.com/mongodb/jasper/options"
+	"github.com/pkg/errors"
+)
+
+func intOK(ok bool) int {
+	if ok {
+		return 1
+	}
+	return 0
+}
+
+// ErrorResponse represents a response indicating whether the operation was okay
+// and errors, if any.
+type ErrorResponse struct {
```

Comment:
seems like most of the reponse building code should end up in mrpc)

Reply:
I don't think this makes sense given that all of the payloads are Jasper-specific and not defined in the mongo wire protocol spec.